recipes/eglibc/eglibc_2.18-r24943.oe.sig

```diff
@@ -1 +1,2 @@
+517cfd34b8d250dcdca0f3490fa4e9bf571ee473  svn://www.eglibc.org/branches/eglibc-2_18;rev=r24943
```

Comment:
Wow, this is odd.

When checking out with svn protocol, you get one additional file compared to checkout with http:
linuxthreads/linuxthreads/manager.c

Any ideas for how this makes sense?